### PR TITLE
fix(kuma-cp): allow to set policies order from others projects

### PR DESCRIPTION
**Pull request by lukidzi**
If someone inherits Kuma and wants to create a new policy and set it in a specific order that wasn't possible. Added function that allows setting order of the policies.
- [X] Link to docs PR or issue --
- [X] Link to UI issue or PR --
- [X] Is the [issue worked on linked][1]? --
- [X] The PR does not hardcode values that might break projects that depend on kuma (e.g. "kumahq" as a image registry) -- fixes possibility to set from other project
- [X] The PR will work for both Linux and Windows, system specific functions like `syscall.Mkfifo` have equivalent implementation on the other OS --
- [X] Unit Tests --
- [X] E2E Tests --
- [X] Manual Universal Tests --
- [X] Manual Kubernetes Tests --
- [X] Do you need to update [`UPGRADE.md`](../blob/master/UPGRADE.md)? --
- [X] Does it need to be backported according to the [backporting policy](../blob/master/CONTRIBUTING.md#backporting)? --
- [X] Do you need to explicitly set a [`> Changelog:` entry here](../blob/master/CONTRIBUTING.md#submitting-a-patch) or add a `ci/` label to run fewer/more tests?

[1]: https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

### CODE REVIEW

File: pkg/plugins/policies/policies.go [hunk 1]
```diff
@@ -22,3 +22,7 @@ var policies = []plugins.PluginName{
 func Enabled() []plugins.PluginName {
 	return policies
 }
+
+func SetPolicies(newPolicies []plugins.PluginName) {
```
Comment (lukidzi) [inline pkg/plugins/policies/policies.go:26]: We can make extend this by code or by configuration. I think we agreed that we don't want to expose the configuration but maybe it's worth it if there is another plugin in the repository extending the current.

Comment (michaelbeaumont) [reply pkg/plugins/policies/policies.go:26]: > I think we agreed that we don't want to expose the configuration but maybe it's worth it if there is another plugin in the repository extending the current.

Do you mean it's worth doing more than this PR does?

Comment (jakubdyszkiewicz) [reply pkg/plugins/policies/policies.go:26]: why not just make te var public and ditch the `Enabled()` and `SetPolicies()`

Comment (lukidzi) [reply pkg/plugins/policies/policies.go]: > > I think we agreed that we don't want to expose the configuration but maybe it's worth it if there is another plugin in the repository extending the current.
> 
> Do you mean it's worth doing more than this PR does?

I thought that maybe doing configuration would be easier for other projects but even when the code change is required it doesn't seem to be a huge change.

**Review by jakubdyszkiewicz — APPROVED** [review]

**Pull request merged**